Home brewed.
This is to fix info syntax for four e-mail addresses.
Will be sent up-stream.

--- screen-4.9.0/doc/screen.texinfo.orig	2022-02-11 11:43:05.112219388 +0100
+++ screen-4.9.0/doc/screen.texinfo	2022-02-11 11:43:32.380572364 +0100
@@ -5863,10 +5863,10 @@
 ============
 
 @verbatim
-     Maarten ter Huurne <maarten@treewalker.org>,
-     Jussi Kukkonen <jussi.kukkonen@intel.com>,
-     Thomas Renninger <treen@suse.com>,
-     Axel Beckert <abe@deuxchevaux.org>,
+     Maarten ter Huurne <maarten@@treewalker.org>,
+     Jussi Kukkonen <jussi.kukkonen@@intel.com>,
+     Thomas Renninger <treen@@suse.com>,
+     Axel Beckert <abe@@deuxchevaux.org>,
      Ken Beal <kbeal@@amber.ssd.csd.harris.com>,
      Rudolf Koenig <rfkoenig@@informatik.uni-erlangen.de>,
      Toerless Eckert <eckert@@informatik.uni-erlangen.de>,
